### DISCUSSION TITLE
Relax mod customization

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#21918] [Ideas]
I want to practice my aim and common Relax can't help me with it properly. 

Relax mod hits an object even when the cursor was on the object for less than 5ms - it's almost impossible to hit in real conditions, thus playing with this mod develops a bad habit.
This problem can be fixed by implementation of timer that count time on object - if time is less than X ms then it should be miss. X variable can be configured in mod customization.

Also with this timer it will be possible to create "early exit" from object (also can be a new customization switch) - if cursor was in object more than X ms but left an object before perfect timing then it should be late hit on edge (of corse only if this late hit still within accuracy range).

These customizations will help to train aim properly, so I hope it will be implemented sometime.

**Comment by Rekunan** [comment]
I think regardless you should be training your aim on non-relax instead as too much relax will mess up your aim-tap coordination.
